backend/views/core/auth/login.py

```diff
@@ -73,9 +75,10 @@ def login_manual(request: HttpRequest):  # HTMX POST
         messages.error(request, "Incorrect email or password")
         return render_toast_message(request)
 
-    if user.awaiting_email_verification and ARE_EMAILS_ENABLED:
-        messages.error(request, "You must verify your email before logging in.")
-        return render_toast_message(request)
+    if request.user.is_authenticated:
```

Comment:
Yes I know but `mypy` still was complaining since user type with `HtmxAnyHttpRequest` is `User | AnonymousUser`. It's the same error that forced us to create `HtmxHttpRequest`. I wanted to avoid using `# type: ignore` that's why I placed this check. I can place it in one `if` statement together with `user.awaiting_email_verification and ARE_EMAILS_ENABLED` to avoid nesting but it's still same thing.

Reply:
Okay cool, yeah that's true. As long as it's not making anything too much worse and there's overall more benefit I guess its fine. Thanks :)

Reply:
You welcome ;). Tomorrow I will change this `if request.user.is_authenticated:` and will use `type: ignore` on that one.